### ISSUE TITLE
the effect on UCF101 dataset

### ISSUE THREAD

**Issue by Kathrine94** [DavideA/c3d-pytorch#9]
Hello! I'm try to use your model to train UCF101, but the accuracy is only 50 % .Have you train the model on UCF101 datasets ？Is the test set accurate to 80% or more?

**Comment by DavideA** [comment]
Hi and thanks for interest.

I did not train this model.
Instead, I carefully ported weights from a previous existing Keras implementation. Such keras implementation, in turn, carefully ported weights from the original Caffe version.

In this Caffe to Keras to Pytorch porting, several things can go wrong, so I'm not sure this model is exactly the same as the original Caffe model.

I did not test its performances, but I successfully used it for some applications a few times.

You can either use it as it is, with pretrained weights, or retrain it from scratch. 
If you retrain it, you should be able to get at least near the results reported in the original paper. Are you following the exact same training protocol as reported by authors?

**Comment by DavideA** [comment]
Moreover, available weights are available only for Sports1M. 

No UCF101, unfortunately :(

**Comment by DavideA** [comment]
Closing.
